tests/units/classes/asserters/castToArray.php

```diff
@@ -47,10 +47,10 @@ public function testSetWith()
                     ->setAdapter($adapter)
             )
             ->then
-                ->object($asserter->setWith($object = new \exception()))->isIdenticalTo($asserter)
+                ->object($asserter->setWith($object = new \arrayiterator()))->isIdenticalTo($asserter)
```

Comment:
are you sure about this?

Reply:
Casting the exception to array was trigerring an error in the PHP 8.2 build I used, so I had to found something that can be casted to array, in order to fix this test.